Source/Csla.Generators/cs/AutoImplementProperties/Csla.Generator.Tests/Snapshots/GeneratorSnapshotTests.ReadOnlyBaseInterface#InterfaceImplementBusinessPOCO.g.verified.cs

```diff
@@ -3,6 +3,7 @@
 #nullable enable
 using System;
```

Comment:
Should this file be in source control? It is generated?

Reply:
in this video https://www.youtube.com/watch?v=azJm_Y2nbAI the author says that is a good practice to put the nullable enable in teh begining

Reply:
I agree with the `#nullable enable` - I am wondering if the `.g.verified.cs` files should be in source control at all - if they are generated on each build they should not be in source control.

Reply:
this is part of the snapshot unit test, when the test runs it makes a .received.cs and compares both files

the verivied is there to check the test result